### PR TITLE
Adds /help as a chat command.

### PR DESCRIPTION
**Pull request by tacobowl8**

### CODE REVIEW

File: Initium-Odp/war/odp/javascript/messager-impl.js [hunk 1]
```diff
@@ -112,6 +112,9 @@ messager.onChatMessage = function(chatMessage)
 			chatMessage.message = "/me is a spammer. Kill him with fire!";
 		var meMode = chatMessage.message.toLowerCase().startsWith("/me ");
 		html+="<span class='chatMessage-text'>";
+		if(chatMessage.message.toLowerCase() == "/help") {
+			chatMessage.message = "<a href=\"#\" onclick=\"pagePopup('/odp/chatHelp.html')\">[Click here to view chat help]</a>";
```
Comment (Emperorlou) [inline Initium-Odp/war/odp/javascript/messager-impl.js]: No I think we should just do the help pop-up immediately and then not send the chat message to the server at all.

Comment (Emperorlou) [inline Initium-Odp/war/odp/javascript/messager-impl.js]: Don't you think?

Comment (Emperorlou) [inline Initium-Odp/war/odp/javascript/messager-impl.js]: If you still want to do it this way I'll have to add it in my end. It's OK whatever you decide you want to do.

**Comment by tacobowl8** [comment]
Put /help to only trigger clientside and not put anything into the chat.  Also added /trade <character name>

File: Initium-Odp/war/odp/javascript/messager-impl.js [hunk 2]
```diff
@@ -57,6 +57,12 @@ messager.onNotificationMessage = function(message)
 
 messager.onChatMessage = function(chatMessage)
 {
+	if (chatMessage.message.toLowerCase().startsWith("/")) {
```
Comment (Emperorlou) [inline Initium-Odp/war/odp/javascript/messager-impl.js]: To lower case unnecessary here. Also......... Are you sure Js has that function for strings?

File: Initium-Odp/war/odp/javascript/messager-impl.js [hunk 3]
```diff
@@ -145,6 +151,18 @@ messager.onChatMessage = function(chatMessage)
 	notifyNewMessage(chatMessage.code);
 };
 
+function tryChatCommands(chatMessage)
+{
+	if (chatMessage.message == "/help") {
```
Comment (Emperorlou) [inline Initium-Odp/war/odp/javascript/messager-impl.js]: Here is where you would want to call to lower case

**Pull request merged**